documentation/adoc/docu.adoc

```diff
@@ -28,6 +28,8 @@ include::cluster-controller.adoc[]
 
 include::topic-controller.adoc[]
 
+include::metrics.adoc[]
```

Comment:
Maybe we want to rename it as appendix_metrics.adoc ? As @scholzj suggested, metrics info are more an appendix and not something fully supported by the project.